cylc/flow/unicode_rules.py

```diff
@@ -351,6 +351,8 @@ class TaskOutputValidator(UnicodeRuleChecker):
         not_starts_with('_cylc'),
         # blacklist keywords
         not_equals('required', 'optional', 'all', 'and', 'or'),
+        # blacklist Run Modes:
+        not_equals(*RunMode.MODES),
```

Comment:
Shouldn't this just be `RunMode.SKIP`?

Reply:
I'm really not sure that it's a good idea to have any of the RunMode strings as custom outputs. But you are right, it might concievably break someone's existing practice.